astro/src/components/docs/release-notes/ThemeUpdateWarning.astro

```diff
@@ -0,0 +1,11 @@
+---
+import Aside from 'src/components/Aside.astro';
+---
+
+<Aside type="theme" title="Theme update">
+<p>The theme has changed and an upgrade is required for this version of FusionAuth. Custom themes will render new pages according to the default theme until a custom version has been added.</p>
+
+<p>See <a href="/docs/customize/look-and-feel/advanced-theme-editor#upgrading">Theme Upgrades</a> for more information about theme upgrades.</p>
+
+<slot/>
```

Comment:
I think we should move the `<slot/>` up above the `See Theme Upgrades for more information...` paragraph. The child content is going to be more relevant (hopefully) to the release in question and more important for the reader to read.

Reply:
I also realized that paragraph read as
> See Theme Upgrades for more information about theme upgrades.

and changed it to
> See Theme Upgrades for more information about the upgrade process.